src/main/java/picard/sam/ValidateSamFile.java

```diff
@@ -205,6 +205,8 @@ protected int doWork() {
                 IOUtil.assertFileIsReadable(REFERENCE_SEQUENCE);
                 reference = ReferenceSequenceFileFactory.getReferenceSequenceFile(REFERENCE_SEQUENCE);
 
+            } else {
+                log.warn("NM validation cannot be performed without the reference");
```

Comment:
is that the only validation that doesn't occur? 

Also, please add a period in the end, and make it clear that all other validations will still happen.

Reply:
I believe that is the only validation that needs a warning if a reference file isn't provided.

I've fixed the warning message based on your comments.